packages/tasks/src/model-libraries.ts

```diff
@@ -446,6 +446,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.pyannote_audio,
 		filter: true,
 	},
+	"py-feat": {
+		prettyLabel: "py-feat",
+		repoName: "py-feat",
+		repoUrl: "https://github.com/cosanlab/py-feat",
+		docsUrl: "https://py-feat.org/",
+		snippets: snippets.py-feat,
+		filter: true,
```

Comment:
For this to work, there should be models in the Hub tagged as being compatible with the library. For instance, https://huggingface.co/py-feat/retinaface would be a candidate.

Reply:
Thanks @pcuenca for the feedback. I think I'm still a little confused about how code snippets work with huggingface. I have tried to include snippets of code for how to use each model in the model cards, but I'm not sure how to add tags to this that can be referenced in the snippets attribute. Maybe it's just best to remove for now?